packages/frontend/pages/provenance/[deviceKey].vue

```diff
@@ -80,28 +80,28 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>. -->
 
             <section ref= "section" id="priority-notices">
               <ProvenancePriorityNotices :deviceKey="deviceKey" :provenance="provenance"/>
-            </section>
+            </section><br>
 
             <section id="recent">
               <ProvenanceFeed :deviceKey="deviceKey" :provenance="provenance_noRecord"/>
             </section>
             <section id="device-creation">
               <ProvenanceFeed :deviceKey="deviceKey" :provenance="deviceCreationRecord"/>
-            </section>
+            </section><br>
             <section id="create-record">
               <ProvenanceCreateRecord :deviceRecord="deviceRecord" :deviceKey="deviceKey"/>
             </section>
-            <section id="child-keys">
+            <section id="child-keys"><br>
```

Comment:
Is it possible to add a CSS class instead of the `<br>` to acheive the desired spacing?